content_script.js

```diff
@@ -7,6 +7,11 @@ var imageFormatExtension = "jpeg";
 
 // Take screenshot
 captureScreenshot = function () {
+  // Check for encrypted content
+  if (document.querySelector('video').mediaKeys != null) {
+    alert("Cannot screenshot copyright-protected content.");
```

Comment:
IMHO, a notification "pop-up" would be nicer. In such a case, a communication with background script should be done, but it is easy to add.